docs/creating_integration_manifest.md

```diff
@@ -51,9 +51,17 @@ The version of the integration is required for custom integrations. The version
 
 ## Integration Type
 
-Define what kind of integration this is. Currently accepted values are `integration` and `helper`. Helpers are integrations that provide entities to help the user with automations like input boolean, derivative or group.
-
-Defaults to `integration` if not set.
+Define what kind of integration this is. Defaults to `hub` if not set.
```

Comment:
Should we mention "main focus" or expand the description for `service` and `hub` to make it clear that they are not exclusive, and can contain devices or services